hivemind/dht/__init__.py

```diff
@@ -274,289 +243,13 @@ async def _get_visible_address(self, node: DHTNode, num_peers: Optional[int], pe
             future.set_exception(ValueError(f"Can't get address: DHT node has no peers and no public endpoint."
                                             f" Please ensure the node is connected or specify peers=... manually."))
 
-    def declare_experts(self, uids: Sequence[ExpertUID], endpoint: Endpoint, wait: bool = True,
-                        timeout: Optional[float] = None) -> Dict[ExpertUID, bool]:
-        """
-        Make experts visible to all DHT peers; update timestamps if declared previously.
+    @deprecated(version='0.9.5', reason="dht.declare_experts is deprecated, please use hivemind.declare_experts.")
+    def declare_experts(self, uids, endpoint, wait: bool = True):
+        from hivemind.client.dht_ops import declare_experts
```

Comment:
Do we introduce circular dependencies if we move this to all other imports?

Reply:
no, we don't
(moved)